plugins/org.jboss.ide.eclipse.freemarker/src/org/jboss/ide/eclipse/freemarker/editor/ReconcilingStrategy.java

```diff
@@ -118,7 +118,8 @@ void reconcile() {
 	 */
 	private ParseException checkTemplateSyntax() {
 		if (fmConfiguration == null) {
-			fmConfiguration = new Configuration(Configuration.getVersion());
+			// see Freemarker configuration; we should use the version we are supporting, not the latest.
+			fmConfiguration = new Configuration(Configuration.VERSION_2_3_31);
```

Comment:
Developer tools sometimes use the current version on purpose. The ERROR can be suppressed like this: `new Configuration(new Version(Configuration.getVersion().toString()))`

Reply:
I can change it but from my understanding, this is more about the version the eclipse plugin use rather than the version the developer use. Since the plugin is using version 2.3.31, this is more about finding issue if the plugin embedded freemarker is updated without this line (and the code) behind updated.
I can change it anyway (I just want to be sure my understanding is good :))